assets/scss/main.scss

```diff
@@ -395,3 +395,7 @@ label.form-label[for="signer"] {
   float:right;
   font-weight:700;
 }
+
+.nowrap {
```

Comment:
This will affect to english language, I think needs to check every page

Reply:
What? How? I only created a new class and applied it to a word in Welsh.

Reply:
And there is no pre-existing `nowrap` class, even in GDS, so no clashes or anything....